blaze/server/tests/test_server.py

```diff
@@ -386,6 +386,44 @@ def test_server_accepts_non_nonzero_ables():
 
 
 @pytest.mark.parametrize('serial', all_formats)
+def test_map_client_server(iris_server, serial):
+    print(serial)
+    test = iris_server
+    t = symbol('t', discover(iris))
+    expr = t.species.map(len, 'int')
+    query = {'expr': to_tree(expr)}
+    response = test.post('/compute',
+                         data=serial.dumps(query),
+                         headers=mimetype(serial))
+    assert 'OK' in response.status
+    respdata = serial.loads(response.data)
+    result = serial.data_loads(respdata['data'])
+
+    istrue = result == compute(expr, {t: iris}, return_type=list)
+    try:
+        assert istrue
+    except:
```

Comment:
The question though: for fastmsgpack - pandas loads data into a Series - which is inconsistent with other serialization types that load a list.

`fastmsgpack_object_hook` is returning Series and DataFrames. Should it return lists to be consistent with other serialization objects. If it doesn't matter then I can break up the test so it does the correct assertion for different serializations.

It seems like all serialization should return consistent results, but I don't know what this effects.